### PR TITLE
(WORKSHOP BOUNTY) Resubmission#3 asked by SAM

### PR DESCRIPTION
**Pull request by HariOm987**
I have corrected all the grammar and phrasing error and inserted server side coding part. Hoping it is ready now. :))

Past [pull request](https://github.com/hackclub/hackclub/pull/1482)

### CODE REVIEW

**Comment by HariOm987** [comment]
@sampoder I have made all the changes asked by you. Please take a look.

File: password-generator/README.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+2-	We customized it to make it look more attractive.
+
+Now with JavaScript, we will add functionality to our tool and will set up a password to the website.
+
+## 3) JavaScript
+Navigate to `script.js` file.
+
+### 1- Adding a password to the website.
+#### Disclaimer: This isn't the greatest method of securing a web app as password is visible in source code.
```
Comment (sampoder) [inline password-generator/README.md:280]:
```suggestion
**Disclaimer: This isn't the greatest method of securing a web app as password is visible in source code. **
```

File: password-generator/README.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+2-	We customized it to make it look more attractive.
+
+Now with JavaScript, we will add functionality to our tool and will set up a password to the website.
+
+## 3) JavaScript
+Navigate to `script.js` file.
+
+### 1- Adding a password to the website.
+#### Disclaimer: This isn't the greatest method of securing a web app as password is visible in source code.  
+You can try [code obfuscation](https://obfuscator.io/) and [server side coding](https://www.hindawi.com/journals/scn/2019/9629034/) to make it more secure.
```
Comment (sampoder) [inline password-generator/README.md:281]:
```suggestion
You should run all authentication code server side when shipping a final product.
```

**Review by sampoder — APPROVED** [review]
Congrats @HariOm987 I believe this workshop is ready! Great work and thank you for being so flexible :)

**Comment by HariOm987** [comment]
### Oh that's cool.🤩..... @MatthewStanciu Can you please give it a final pass and merge it? 🙌

**Comment by sampoder** [comment]
Seems like this never got put on the workshops site :/

Will handle ASAP.

**Pull request merged**